include/fluidsynth/settings.h

```diff
@@ -48,6 +48,9 @@ extern "C" {
  *       // ...
  *     }
  * @endcode
+ * All string settings are encoded in UTF-8. This includes the names
+ * of the audio and MIDI devices, exposed as setting options.
```

Comment:
Probably it's also worth mentioning in the `fluidsettings.xml`, that starting with 2.3.6 all device names are expected to be UTF8 encoded.

Reply:
done